lib/shared/bucketing-assembly-script/assembly/bucketing/segmentation.ts

```diff
@@ -83,7 +91,11 @@ function filterFunctionsBySubtype(subType: string, user: DVCPopulatedUser, filte
         return _checkStringsFilter(user.platform, filter)
     } else if (subType === 'customData') {
         if (!(filter instanceof CustomDataFilter)) {
-            throw new Error('Invalid filter data')
```

Comment:
I think this would legitimately be an error, it shouldn't be able to reach this point without the filter being a CustomDataFilter. This was mostly here for the benefit of the type system I think

Reply:
will revert this one